php/Terminus/Models/Workflow.php

```diff
@@ -23,6 +23,9 @@ class Workflow extends TerminusModel {
    */
   private $user;
 
+  // @TODO: Make this configurable.
```

Comment:
Yeah, thanks! I am so glad I made a habit of using `git push --force-with-lease` for this very reason. It has saved my skin a few times this week.